pom.xml

```diff
@@ -290,7 +290,7 @@
       <plugins>
         <plugin>
           <artifactId>maven-assembly-plugin</artifactId>
-          <version>3.2.0</version>
+          <version>3.3.0</version>
```

Comment:
@davidB What is maven-assembly-plugin here for?

Reply:
I think, it could be removed. I don't remember why I use it, maybe to create jar with source or please a legacy requirement for central nexus or maven-plugin (too lazy to search in the history)

Reply:
Let's drop it then :)